### PR TITLE
added container cofiguration to the node shcema

### PR DESCRIPTION
**Pull request by engdorm**
add support in container configuration to enable running nodes inside containers

[^note]:
    Put an `x` into the [ ] to show you have filled the information.
    The template comes from https://github.com/MOV-AI/.github/blob/master/.github/pull_request_template.md
    You can override it by creating .github/pull_request_template.md  in your own repository

### CODE REVIEW

**Review by Mograbi — APPROVED** [review]

File: dal/classes/protocols/message_client.py
Comment (Mograbi) [inline dal/classes/protocols/message_client.py:1]: I think we already have that in message-server right ? 
in this case maybe we should move it to shared ?

File: dal/scopes/fleetrobot.py [hunk 1]
```diff
@@ -69,7 +76,7 @@ def remove_alert(self, alert: str) -> None:
         """
         active_alerts = self.get_active_alerts()
         if alert in active_alerts:
-            #active_alerts.pop(alert)
+            # active_alerts.pop(alert)
```
Comment (Mograbi) [inline dal/scopes/fleetrobot.py:79]: remove line

**Comment by engdorm** [comment]
doing a new commit with the updated code
see
https://github.com/MOV-AI/movai-core-shared/pull/58

File: dal/classes/protocols/message_client.py
Comment (engdorm) [reply dal/classes/protocols/message_client.py:1]: will be deleted 
old code

**Pull request closed without merging**